server/api/users.go

```diff
@@ -170,6 +182,7 @@ func (a *API) handleGetMe(w http.ResponseWriter, r *http.Request) {
 		user.Permissions = append(user.Permissions, model.PermissionCreatePost.Id)
 	}
 
+	user.Sanitize(map[string]bool{})
```

Comment:
As the user is fetching themselves, shouldn't we sanitize this profile with all the options set or at least with those coming from the config?

Reply:
as far as I'm aware, this is the same as channels, for the same user we just remove the password and MFA secret